context/AuthContext.js

```diff
@@ -72,9 +77,12 @@ export function AuthProvider({children}) {
 
   const isLoggedIn = async () => {
```

Comment:
could you also add `// only when user logged before and quite app` above `isLoggedIn` function so that we (aka I ) don't get confused about its function?